File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.cs
using System;

public class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int num_list_multiply = 1;
        int num_list_add = 0;
        for (int i=0; i<num_list.Length; i++) {
            num_list_multiply *= num_list[i];
            num_list_add += num_list[i];
        }
        return answer = (num_list_multiply < num_list_add * num_list_add) ? 1 : 0;
    }
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) 
{
    int num_list_multiply = 1;
    int num_list_sum = 0;
    for(int i=0; i<num_list.size(); i++)
    {
        num_list_multiply *= num_list[i];
        num_list_sum += num_list[i];
    }
    return (num_list_multiply < num_list_sum * num_list_sum) ? 1 : 0;
}